pages/_document.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import Document, { DocumentContext, Head, Html, Main, NextScript } from "next/document";
+import { NextSeo } from "next-seo";
```

Comment:
I don't see it used. Did you try to do https://github.com/garmeeh/next-seo#default-seo-configuration ?

Reply:
missed removing this out. I don't think we need a default SEO file actually

Reply:
ok added default